budgetkey_data_pipelines/pipelines/procurement/tenders/exemptions/pipeline-spec.yaml

```diff
@@ -1,34 +1,69 @@
-publishers:
+exemptions:
   pipeline:
     - run: add_metadata
       parameters:
-        name: "procurement-tenders-exemptions-publishers"
+        name: "procurement-tenders-exemptions"
+    # get the main HTML page of the exemptions search
     - run: add_resource
       parameters:
         name: "mr-gov-il-search-exemption-messages"
         url: "http://www.mr.gov.il/ExemptionMessage/Pages/SearchExemptionMessages.aspx"
         format: "txt"
+    # download it (using tzabar runner which does some magic that bypasses network security)
     - run: stream_remote_resources
       runner: tzabar
+    # parse all the publisher ids from the main search page
     - run: add_exemptions_publishers_resource
+    # go over each publisher and get urls for all exemptions related to this publisher
+    # max_pages parameter can be used to limit number of pages for each publisher
+    - run: add_publisher_urls_resource
+      runner: tzabar
+      parameters:
+        # meant for running in cronjob every day
+        # for a one-time download of all data set environment variable: EXEMPTION_PUBLISHERS_MAX_PAGES=-1
+        max_pages: 5
+    # download the data from each exemption page
+    - run: download_exemption_pages_data
+    # parse the data for each exemption
+    - run: parse_exemption_data
     - run: dump.to_path
       parameters:
-        out-path: /var/datapackages/procurement/tenders/exemptions/publishers/
+        out-path: /var/datapackages/procurement/tenders/exemptions
+    # remove the documents json field before updating SQL
+    - run: set_types
+      parameters:
+        resources:
+          exemptions
+        types:
+          documents_json: null
+    # update the SQL table (based on the primary keys defined in the schemas)
+    - run: dump.to_sql
+      parameters:
+        tables:
+          procurement_tenders_exemptions:
+            mode: update
+            resource-name: exemptions
 
-publisher-urls:
-  dependencies:
-    - pipeline: ./procurement/tenders/exemptions/publishers
+exemption-documents:
```

Comment:
After reading the code, I don't think we actually need this pipeline - as all exemption document data is attached as a nested object in the main exemption record.